docs/index.md

```diff
@@ -1,91 +1,77 @@
 # Getting Started with Mirascope
 
-Welcome to Mirascope! This guide will help you dive into building powerful LLM-powered applications with ease. We'll start with an interactive tutorial and then guide you through next steps to deepen your knowledge and understanding of Mirascope and all of its features.
+Mirascope: Build powerful LLM applications with ease.
 
-## Interactive Web Search Agent Tutorial
+## Quick Start
 
-Jump right in with our hands-on tutorial where you'll interact with a web search agent built using Mirascope. This will give you a practical taste of Mirascope's key features.
+Install Mirascope and set up your API key:
 
-### Replit Environment
+```bash
+pip install "mirascope[openai]"  # For OpenAI support
+pip install "mirascope[anthropic]"  # For Anthropic support
+# For other providers, use: pip install "mirascope[provider_name]"
+```
 
-We've set up a Replit environment for you to start immediately without any local setup:
+```python
+import os
+os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"
+# For Anthropic: os.environ["ANTHROPIC_API_KEY"] = "YOUR_API_KEY"
+```
+Mirascope supports various LLM providers, including [OpenAI](https://openai.com/), [Anthropic](https://www.anthropic.com/), [Mistral](https://mistral.ai/), [Gemini](https://gemini.google.com), [Groq](https://groq.com/), [Cohere](https://cohere.com/), [LiteLLM](https://www.litellm.ai/), [Azure AI](https://azure.microsoft.com/en-us/solutions/ai), and [Vertex AI](https://cloud.google.com/vertex-ai).
 
-[Mirascope Getting Started Tutorial](https://replit.com/@willbakst/Mirascope-Getting-Started-Tutorial?v=1)
 
-### How to Use the Replit
+Create your first LLM-powered function:
```

Comment:
i like this section, but to keep things matching the book recommendation examples, use "Recommend a {genre} book" here please